Riot/Modules/Room/Attachements/MXKAttachmentsViewController.m

```diff
@@ -1379,6 +1379,28 @@ - (void)documentInteractionControllerDidDismissOpenInMenu:(UIDocumentInteraction
     }
 }
 
+#pragma mark - MXKDestinationAttachmentAnimatorDelegate
+
+- (BOOL)prepareSubviewsForTransition:(BOOL)isStartInteraction
+{
+    MXKMediaCollectionViewCell *cell = (MXKMediaCollectionViewCell *)[self.attachmentsCollection.visibleCells firstObject];
+    MXKAttachment *attachment = attachments[currentVisibleItemIndex];
+    NSString *mimeType = attachment.contentInfo[@"mimetype"];
+
+    // Check attachment type for GIFs - this is required because of the extra WKWebView
+    if (attachment.type == MXKAttachmentTypeImage && attachment.contentURL && [mimeType isEqualToString:@"image/gif"])
+    {
+        UIView *customView = cell.customView;
+        for (UIView * v in customView.subviews) {
```

Comment:
2 minor nits:
- There's a small whitespace issue here.
- Our Objective-c style is to use a new line for the `{`.
```suggestion
        for (UIView *v in customView.subviews)
        {
```